terraso_backend/apps/graphql/schema/sites.py

```diff
@@ -187,25 +195,36 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         log = cls.get_logger()
         user = info.context.user
         site = cls.get_or_throw(Site, "id", kwargs["id"])
-        if not user.has_perm(Site.get_perm("change"), site):
-            raise cls.not_allowed(MutationTypes.UPDATE)
 
-        # if any site settings fields are present in the mutuation, check that the user
```

Comment:
The "settings" distinction was introduced in a prior PR of mine; however, further review indicated that all properties which are changeable via this request are in fact "settings" which should be guarded by that permission. (There is no use-case for, e.g., a contributor changing a site's lat-lon.)